### DISCUSSION TITLE
Has anyone used the 6 pin Molex connector from the MobileLink LTE dongle?

### DISCUSSION THREAD

**Discussion by SRFurst** [jgyates/genmon#1077] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open issues](https://github.com/jgyates/genmon/issues),[closed issues](https://github.com/jgyates/genmon/issues?q=is%3Aissue+is%3Aclosed),[open discussion](https://github.com/jgyates/genmon/discussions), [closed discussion](https://github.com/jgyates/genmon/discussions/categories/questions?discussions_q=is%3Aclosed) - which may answer my question or that I could contribute my information to.
- [X] I have read the [Known Issues Wiki Page](https://github.com/jgyates/genmon/wiki/Appendix-D-Known-Issues) and my question / issue is not listed.
- [X] I have provided the version of the software and the operating system in my post.
- [X] If I have an issue with genmon not communicating with the controller, I have read and followed the [serial troubleshooting guide](https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting).

### Have you submitted your log files from the About page? Note that outbound email must be working for logs to be submitted.

I have not submitted logs

### What is/are your question(s)?

When I purchased my generator (22kW air cooled) it came with a year of LTE monitoring. The cellular dongle attaches via a 5 wire cable connected to a six-pin Molex connector. Two of the wires (red and black) are connected to the generator battery and two (a white and a black) originate from the 8-pin connector on the generator controller. The fifth (green) doesn't seem to do anything.

I assume the white/black pair are the RS-232 Rx/Tx signals. This cable seems pretty convenient -- has anyone tried to use it?

**Comment by skipfire** [comment]
I assume you are asking about cutting the end of and splicing onto it to make your own cable which as 2 areas to consider, power and data.

Looking at the data part, as long as it is using RS232 (which I believe the cellular does but you can post a picture of the plug to be sure) then it should work, but of course you are damaging MobileLink device.  I would suggest just buying the cheap short 8-pin cable on [Amazon](https://www.amazon.com/gp/product/B000M802RG) (only $7) rather than splicing off the MobileLink 6-pin.

As for power, you would need to convert the 12v power to 5v and/or 3v3 for the TTL and Pi but the better options here are going to attach to the battery terminals not to the Molex cable.

**Comment by rsockanc** [comment]
It can work. I had the wifi add-on box that was a joke in terms of functionality so I used the Molex it came with by cutting it off and splicing in some Viper clamps then running the serial connection over cat5 to a RJ45 to USB convertor so my PI is sitting inside the house on its own UPS.
![IMG_2420](https://github.com/jgyates/genmon/assets/24508741/5972a6c2-355e-4b16-9051-410ca8e187ac)
![IMG_2399](https://github.com/jgyates/genmon/assets/24508741/b4bcfe33-4bf4-4f8a-a8d7-d68d9e69edad)
![IMG_2417](https://github.com/jgyates/genmon/assets/24508741/1b27f36b-d9f8-41db-9f8b-4c7e6b05b2a6)
![IMG_2400](https://github.com/jgyates/genmon/assets/24508741/49dd4ec7-8145-4ee7-927c-218a4a7ced53)
![IMG_2404](https://github.com/jgyates/genmon/assets/24508741/34d5bf90-506b-4f5c-a16f-701171ea0241)
![IMG_2413](https://github.com/jgyates/genmon/assets/24508741/c3531ebd-4768-4143-8c4f-060f3d57db80)
![IMG_2415](https://github.com/jgyates/genmon/assets/24508741/33380355-3222-4b3d-9271-5960e7ab5a1d)
![IMG_2416](https://github.com/jgyates/genmon/assets/24508741/3cbd2ff6-1011-4e80-a38a-29cb3d5a7ab8)
![IMG_2418](https://github.com/jgyates/genmon/assets/24508741/abc409e1-5e98-42b7-a237-da7a13d9bca6)
![IMG_2421](https://github.com/jgyates/genmon/assets/24508741/2cebeceb-fb53-4f08-a7dd-6c49352ef5d2)
![IMG_2423](https://github.com/jgyates/genmon/assets/24508741/87d5e5f1-7919-4ae9-870c-64548ae49fe6)
